### ISSUE TITLE
Training hyperparameters for Re500

### ISSUE THREAD

**Issue by cfos3120** [neuraloperator/markov_neural_operator#4]
I have successfully recreated the Re=40 results for Navier-Stokes, using the script defaults in the repository. But extending this to Re500, (with the correct Radii) at 900-100 training data split. Does not yield the same accuracy as in the paper:

`Epoch 50 completed in 359.699 seconds. Train err: 0.330 Test L2 err: 0.166 Test H1 err: 0.493 Test H2 err: 0.702 Train diss err: 0.031`

Were different parameters used for training?

**Comment by mliuschi** [comment]
Hi @cfos3120, I'm assuming you're looking at Table 4 in the paper? I believe this represents $$L^2$$ errors for various training and testing regimes. If you're training and testing on vorticity, then your test $$L^2$$ error (0.166) seems to be close to those  in the Table (0.1736, 0.1694, 0.1854, depending on which loss you're using for training), if I'm not mistaken. Please let me know if I'm looking in the wrong place.
